RELEASING.md

```diff
@@ -107,27 +119,12 @@ Replace the old version with the new version in the `master` branches of the fol
     - add a new section with all relevant upgrade details: https://sourcegraph.com/github.com/sourcegraph/sourcegraph@a718276cbdc4c9e079d5495cb34ce663c5d35c01/-/blob/doc/admin/updates/docker_compose.md#updating-a-docker-compose-sourcegraph-instance
     - Update `latestReleaseDockerComposeOrPureDocker` in https://github.com/sourcegraph/sourcegraph/blob/master/cmd/frontend/internal/app/pkg/updatecheck/handler.go#L47
 
-## 8) Post that you are done.
-
-Write a message to #dev-announce:
-
-> Docker Compose v3.9.2 has been released.
-
-## 9) Message @stephen on Slack
+### Message @stephen on Slack
```

Comment:
I went ahead and removed this section calling out to me to follow these steps, since others (primarily Uwe and Dave) will now be doing this. If you could update the upstream release checklist to include a step to consider doing a pure-docker release (e.g. in the case of a major release, but not strictly all patch releases) that'd be excellent!

Reply:
Noted here: https://github.com/sourcegraph/sourcegraph/pull/15281/commits/d8bcb6a9a8aa822b55c7fee1a3c2e58f8fb2d92b